include/zephyr/senss/senss_datatypes.h

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef SENSS_DATATYPES_H_
+#define SENSS_DATATYPES_H_
+
+/**
+ * @brief Data Types
+ * @addtogroup senss_datatypes
+ * @{
+ */
+
+/**
+ * @def SENSS_SENSOR_FLAG_PHYSICAL
+ * @brief physical sensor flag
+ */
+#define SENSS_SENSOR_FLAG_PHYSICAL BIT(0);
+
+/**
+ * @def SENSS_SENSOR_FLAG_VIRTUAL
+ * @brief virtual sensor flag
+ */
+#define SENSS_SENSOR_FLAG_VIRTUAL  BIT(1);
```

Comment:
Physical sensor or virtual sensor just sensor subsystem internal concepts,  Application should not aware about this.

Removed the these definition,  since sensor subsystem can know a sensor is virtual or physical during it's registration base on if the sensor registered reporters or not.  If a sensor has reporter(s), then its a virtual sensor.
 
See the sensor register info structure and register API: 
https://github.com/zephyrproject-rtos/zephyr/pull/45370/commits/5a0983bf0e95252602b84713285a25d34d420925#diff-f3d688df79b78cd7f5ca2e8e2a668c576137f5b128ef9f0c6c342333727eb8c5R64